cime/src/drivers/mct/main/seq_rest_mod.F90

```diff
@@ -221,6 +226,15 @@ subroutine seq_rest_read(rest_file, infodata, &
           call seq_io_read(rest_file, gsmap, l2gacc_lx, 'l2gacc_lx')
           call seq_io_read(rest_file, l2gacc_lx_cnt ,'l2gacc_lx_cnt')
        end if
+
+       if (glc_present) then
+          gsmap         => component_get_gsmap_cx(glc(1))
+          x2gacc_gx     => prep_glc_get_x2gacc_gx()
```

Comment:
This variable is only allocated in `prep_glc_mod` if `ocn_c2_glc` is true. Should this block of code be inside a `ocn_c2_glc` conditional?

Reply:
@billsacks , yes, I think you are correct here and below.  However, ocn_c2_glc is a module variable in cime_comp_mod, so it would have to be added to the calling signature for both `seq_rest_read` and `seq_rest_write`.  This would break with the convention of those routine calling signatures, but I don't see a good alternative.  Locally duplicating the logic defining ocn_c2_glc from the info in `infodata` would not be a good idea.  I  am fine with adding ocn_c2_glc to the calls to `seq_rest_read` and `seq_rest_write`, but I wanted to see if that seemed acceptable.

cc: @rljacob